mobidam-sst-management-frontend/frontend/src/components/AddPersonDialog.vue

```diff
@@ -61,12 +61,13 @@
                     <v-text-field
```

Comment:
"Benutzerkennung" instead of "Person angeben"

Reply:
✅